src/pyk/proof/reachability.py

```diff
@@ -53,6 +54,7 @@ class APRProof(Proof, KCFGExploration):
     logs: dict[int, tuple[LogEntry, ...]]
     circularity: bool
     failure_info: APRFailureInfo | None
+    execution_time: float
```

Comment:
Let's make this private. Also, consider a shorter name:

```suggestion
    _exec_time: float
```

Reply:
You'll then need a way to read the value, for that you can add a read-only property:

```python
def exec_time(self) -> float:
    return self._exec_time
```

Reply:
Makes sense. Thank you!